libse/SubtitleFormats/SubtitleFormat.cs

```diff
@@ -174,7 +174,7 @@ public static IEnumerable<SubtitleFormat> AllSubtitleFormats
                     new UnknownSubtitle13(),
                     new UnknownSubtitle14(),
                     new UnknownSubtitle15(),
-                    new UnknownSubtitle16(),
+                    new CavenaCIP(),

```

Comment:
This is a sorted list... and perhaps "UnknowSubtitle83" (the last unknown) should be renamed to UnknownSubtitle16 ?